libnd4j/include/ops/declarable/platform/vednn/veda_vednn.vcpp

```diff
@@ -1,45 +1,194 @@
+#include <stdint.h>
```

Comment:
Mind explaining what vecpp is and ensure it's documented in the readme? My best guess is some something like a configure.in that we manipulate with some parameters?

Reply:
its the extension introduced by veda developers to diferentitate it with cpp in cmake. lets say its kinda .cu* like extension